app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/splash/SplashViewModel.kt

```diff
@@ -41,7 +44,11 @@ class SplashViewModel @Inject constructor(
     fun startTimerToNavigateToLogin(splashTime: Long) {
         viewModelScope.launch(ioDispatcher) {
             delay(splashTime)
-            _shouldNavigateToLogin.value = true
+            if (preferenceManager.getBooleanData(PREF_LOGGED_IN)) {
+                _isLoginSuccess.value = true
+            } else {
+                _shouldNavigateToLogin.value = true
```

Comment:
Could we combine them into a single property - `isLoginSuccess`. The Fragment will observer only `isLoginSuccess`. If it's false, it'll navigate to the login screen. 🤔

Reply:
It controls the time to animate to login. So I cannot combine them into one.